src/utils/codeowners.ts

```diff
@@ -68,7 +68,15 @@ export const createOwnersFile = async (
     content = ownerRules.map((rule) => rulesBlockTemplate(rule.filePath, [`${rule.glob} ${rule.owners.join(' ')}`]));
   }
 
-  fs.writeFileSync(outputFile, contentTemplate(content.join('\n'), originalContent, customRegenerationCommand));
+  const normalizedContent = contentTemplate(
+    content.join('\n'),
+    filterGeneratedContent(originalContent),
+    customRegenerationCommand
+  );
+
+  fs.writeFileSync(outputFile, normalizedContent);
```

Comment:
Yea, I was thinking the same thing when writing it, felt like a side effect

Reply:
For this change, I will change the create to a generate function and leave the actual `fs.writeSync` in the generate command itself.  The 'generate' will happen at the top level instead of the function createCodeowner (renamed generateCodeowner)

Reply:
dealing with mocks that are not sticking at the moment.